### ISSUE TITLE
Point club:workshop is rendered as house number not icon in standard layer

### ISSUE THREAD

**Issue by PeterOSM** [gravitystorm/openstreetmap-carto#4973]
### Expected behavior
I expect a club to appear as the icon visible in the editor, shown in the attached picture. I experimented a little with tags but the problem seems to be related to the club item.

This is the point: https://www.openstreetmap.org/node/11923567693

#### Request
Render club with an icon and name in standard layer.

### Actual behavior
The club appears as a house number from the address tag. The nearby atelier is rendered similarly. The point is visible only in some layers. In standard it is not consistently visible.

### Screenshots with links illustrating the problem

1. Tags   
<img width="384" alt="Screenshot 2024-05-27 at 11 00 50" src="https://github.com/gravitystorm/openstreetmap-carto/assets/170932568/d5b14b62-dce3-464d-b68c-f14d5add54c5">

2. Point in editor rendering    
<img width="702" alt="Screenshot 2024-05-27 at 11 00 03" src="https://github.com/gravitystorm/openstreetmap-carto/assets/170932568/cbbc3c38-e8ff-4cf2-ac8b-b0d1dfa42f62">

3. Standard layer rendering  
<img width="827" alt="Screenshot 2024-05-27 at 11 01 39" src="https://github.com/gravitystorm/openstreetmap-carto/assets/170932568/280ca77b-f06c-4b2a-b7bb-452daff6f402">

4. CyclOSM layer rendering  
<img width="827" alt="Screenshot 2024-05-27 at 11 03 24" src="https://github.com/gravitystorm/openstreetmap-carto/assets/170932568/e4d3c97c-c347-4722-ad50-a0dffd76af3a">

**Comment by HolgerJeromin** [comment]
Do you want to add rendering for 
https://wiki.openstreetmap.org/wiki/Tag:amenity=workshop (548 uses)
or
club=workshop  (exact [ONE](https://taginfo.openstreetmap.org/tags/club=workshop#overview), your example)

**Comment by PeterOSM** [comment]
Thanks for the reply Holger. I think the club tag generally should be rendered for all values with no more specific icon as it appears in the editor properties. As an alternative amenity=workshop could be rendered.

Peter Lancashire
Düsseldorf

**Comment by HolgerJeromin** [comment]
>  I think the club tag generally should be rendered for all values with no more specific icon as it appears in the editor properties. 

That would be issue #4489 which was rejected for [good reasons](https://github.com/gravitystorm/openstreetmap-carto/issues/4489#issuecomment-965022103).

> As an alternative amenity=workshop could be rendered.

This has right now far too few usages.

**Comment by PeterOSM** [comment]
Hello Jerome,

Thanks for your help. Having read issue #4489 <https://github.com/gravitystorm/openstreetmap-carto/issues/4489> I now understand. 

I will use polarbearing's suggestion: For the physical location, there are rendered tagging schemes such as amenity=community_centre + community_centre=club_home for the place where people meet.

On the wiki page for /key/ club I overlooked the entirely blank column headed "Image". A random sample of a few other wiki pages showed that the tag that appears at the top of the iD properties editor is NOT always the one that defines the icon: example coffee shop which is amenity=cafe but cuisine=coffee_shop at the top. The presence of an icon here is misleading in some cases (mine). Some usability engineering is missing here. The editor should show the OSM Carto icon for the currently entered keys and tags.

The wiki does show the OSM Carto icons but only when they exist. When they do not nothing is shown, not even the heading or a place-holder. You must know to look there. I did not. How could this usability error in the wiki be fixed? Sorry, I am new. I am trying to help. I have experience in usability and documentation.

Peter Lancashire
Düsseldorf

**Comment by imagico** [comment]
This touches a few matters previously discussed here:

* rendering of `club=*` - #4489
* rendering of `amenity=hackerspace` - #451
* rendering of `amenity=community_centre` - added in #1744, could be further differentiated by `community_centre=*` (22k uses).
* rendering of `amenity=workshop` - not discussed specifically so far but low use and questionable relevance for our style based on current use.

You are right that the overall use of graphical symbology across projects relevant for mapping (in particular editors, tag documentation and OSM-Carto) is lacking in consistency.  This has a number of different reasons.

* OpenStreetMap is a highly decentralized project and there is no mechanism to enforce compliance to a uniform design standard, not even among different editors for example.  This is widely consider a highly desirable situation (decentralization supporting diversity and innovation, making the project as a whole more robust and more able to adapt) - despite the negative aspects.
* Maps and editors represent - in terms of graphics design as well as in the function symbols have - very different use cases with different constraints regarding symbol sizes, use of colors etc.  This makes development of common symbology difficult.
* Editor developers have - in general - different goals and interests than we have in OSM-Carto.  This to some extent stems from the different function of the respective projects but also from their different histories and the different personal backgrounds and different economical situations of the project maintainers and developers.

That being said we try to look at what symbols are used elsewhere for the tags we render when adding or changing rendering.  But this is primarily for inspiration, consistency in graphical representation with editors is not commonly a point we consider (for the reasons mentioned above).  If you see concrete cases where this could be improved without sacrificing our goals we'd be interested in pointers.  Simply documenting better how different editors symbolize different tags would also be valuable (not only for OSM-Carto but for OSM in general).

**Comment by PeterOSM** [comment]
Hello Christoph,

Thank you for the explanation of the inconsistencies. I have experienced such things in my paid work.

I will take some time to understand how to edit the wiki, and then write a short explanation of the above discussion in the club tag page.

In my particular case I used amenity=community_centre for the point within the building (front door), which describes the physical world and links to the club. It is rendered with an icon, not the building number.

How do I or we close a case / question?

With best wishes
Peter Lancashire
